external/fv3fit/fv3fit/reservoir/model.py

```diff
@@ -194,7 +207,9 @@ def load(cls, path: str) -> "ReservoirComputingModel":
             rank_divider = None
 
         if fs.exists(os.path.join(path, cls._AUTOENCODER_SUBDIR)):
-            autoencoder = Autoencoder.load(os.path.join(path, cls._AUTOENCODER_SUBDIR))
+            autoencoder: ReloadableTransfomer = _load_transformer(
```

Comment:
Mypy was complaining because `fv3fit.load` can return any `Reloadable`, but the autoencoder arg should only be a sklearn transform or dense autoencoder. I figured this was a sign I should have some kind of enforcement on what could get loaded as the autoencoder arg.

Reply:
No need for the type hint if you specify the `_load_transformer` type, right?